laws/src/main/scala/cats/laws/discipline/Arbitrary.scala

```diff
@@ -336,3 +338,29 @@ private[discipline] sealed trait ArbitraryInstances0 {
   implicit def catsLawsArbitraryForCokleisli[F[_], A, B](implicit AFA: Arbitrary[F[A]], CFA: Cogen[F[A]], B: Arbitrary[B]): Arbitrary[Cokleisli[F, A, B]] =
     Arbitrary(Arbitrary.arbitrary[F[A] => B].map(Cokleisli(_)))
 }
+
+private[discipline] sealed trait ArbitraryInstances1 {
+
+  implicit def catsLawsArbitraryForSemigroupOfOption[A](implicit ev: Semigroup[A]): Arbitrary[Semigroup[Option[A]]] =
```

Comment:
I know that I mentioned `Semigroup` as an example of an `Invariant` before, but now that I see this I realize that it's awkward (and inefficient) to test. I think that it would simplify things if you use `ListWrapper` like we are using it elsewhere to grab an instance with only an `Invariant` instance. Sorry for leading you in the wrong direction on this one.

Reply:
I will make the change. Not at all, thanks for helping me.